src/freenas/etc/netcli

```diff
@@ -1597,7 +1599,17 @@ if __name__ == '__main__':
                 main_menu()
             else:
                 show_ip()
-                os.execv('/usr/libexec/getty', ['/usr/libexec/getty', 'Pc'])
+                if IS_LINUX:
+                    proc = subprocess.Popen(['/usr/bin/tty'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+                    stdout, stderr = proc.communicate()
+                    if proc.returncode:
+                        print(f'Failed to determine active tty: {stderr.decode()}')
+                        os.execv('/bin/login', ['/bin/login'])
+                    else:
+                        terminal = stdout.decode().strip().strip('/dev/')
```

Comment:
`strip('/dev/')` is not intended to be used like this and while in this case it works, sometime in the future it might let you down